packages/devtools/lib/src/utils.dart

```diff
@@ -65,6 +65,20 @@ String msAsText(num milliseconds, {bool includeUnit = true}) {
   return '${milliseconds.toStringAsFixed(1)}${includeUnit ? ' ms' : ''}';
 }
 
+num nullSafeMin(num possiblyNullA, num b) {
```

Comment:
nit: to simplify the api, change to 
  if (a == null || b == null) {
    return a ?? b;
  }
that way you don't have to explain that a can be null but not b.

Reply:
Okay, wasn't sure we still wanted to return null if both were null. Will change.

Reply:
it could be reasonable to assert that
a != null || b != null
but probably doesn't really matter.